test/utils/general.test.js

```diff
@@ -24,4 +24,32 @@ describe('prettyPrintJSON', () => {
     expect(output).to.equal('{\n  "a": true,\n  "b": {\n    "c": 1,\n    "d": "2"\n  }\n}')
   })
 })
-  
\ No newline at end of file
+
+describe('from', () => {
```

Comment:
Not sure I understand the naming or purpose of `from`.
Does 
```js
const [one, two] = from(1)(double, treble)
```
...offer something better than...
```js
const one = double(1)
const two = treble(1)
```
This may just be a side note, since `from` wasn't introduced in this PR.

Reply:
Yeah I originally thought it was similar to `pipe`, I'll check for usages of it and see if it can be removed.